safe/utilities/test/test_metadata.py

```diff
@@ -34,7 +35,7 @@ def test_write_iso19115_metadata(self):
         keywords = {
             'date': '26-03-2015 14:03',
             'exposure': 'structure',
-            'keyword_version': '3.2',
+            'keyword_version': '',
```

Comment:
It was meant to be inasafe_keyword_version like the other two in the same file. Thanks for pointing this.